app/scripts/modules/netflix/fastProperties/domain/acaTaskStage.ts

```diff
@@ -13,13 +13,11 @@ export class AcaTaskStage implements IStage {
   public comments: string;
   public canary: Canary;
 
-  constructor(private user: IUser, private command: PropertyCommand, private configDetails: AcaTaskStageConfigDetails, private previousStage?: IStage) {
```

Comment:
because they were `private` and not being used locally.  In Typescript land, access modifiers on ctor args create a local class reference and this was flagged by the linters since they weren't being referenced anywhere.